src/common_runtime/interface/configuration.h

```diff
@@ -0,0 +1,21 @@
+# pragma once
```

Comment:
following the Google C++ style guide https://google.github.io/styleguide/cppguide.html, we don't use `#pragma once` and instead use include guards named relative to the top of the project, so in the case it would be

```
#ifndef COMMON_RUNTIME_INTERFACE_CONFIGURATION_H
#define COMMON_RUNTIME_INTERFACE_CONFIGURATION_H

...

#endif  // COMMON_RUNTIME_INTERFACE_CONFIGURATION_H
```

This comment applies in other places as well.